cmd/root.go

```diff
@@ -26,11 +26,11 @@ var (
 // rootCmd represents the base command when called without any subcommands
 var rootCmd = &cobra.Command{
 	Use:   "deck",
-	Short: "Administer your Kong declaratively",
-	Long: `decK helps you manage Kong clusters with a declarative
+	Short: "Administer your Kong clusters declaratively",
+	Long: `The deck command helps you manage Kong clusters with a declarative
```

Comment:
Is it "the deck command"? Or is this referring to the decK tool? I'm inclined to think the latter.